ui/src/testing/stubs/workspaces-api-stub.ts

```diff
@@ -197,7 +197,7 @@ export class WorkspacesApiStub extends WorkspacesApi {
   ): Promise<WorkspaceUserRolesResponse> {
     return new Promise<WorkspaceUserRolesResponse>((resolve) => {
       const newEtag = fp.defaults(2, body.workspaceEtag + 1);
-      const newItems = fp.defaults([], body.items);
```

Comment:
This was returning an object rather than an array.

Reply:
newEtag looks broken here too